doc/modbus_rtu_set_suppress_echo.txt

```diff
@@ -0,0 +1,50 @@
+modbus_rtu_set_suppress_echo(3)
+===============================
+
+
+NAME
+----
+modbus_rtu_set_suppress_echo - enable/disable local echo suppression
+
+
+SYNOPSIS
+--------
+*int modbus_rtu_set_suppress_echo(modbus_t *'ctx', int 'on');*
+
+
+DESCRIPTION
+-----------
+
+The _modbus_rtu_set_suppress_echo()_ function shall enable or disable
+local echo suppression workaround of the libmodbus context 'ctx'.
+
+On buggy hardware which locally echos all bytes which are transmitted,
```

Comment:
I'd reword this, as I commented earlier, it's not necessarily buggy, there's even a few protocols for multimaster that use this very deliberately.  (not modbus, but it's not _buggy_)

Reply:
I got it and I agree that this could be useful. So I'll remove my ... let's say emotional :grin: ... wording in the version.